tianshou/evaluation/launcher.py

```diff
@@ -26,19 +27,89 @@ class JoblibConfig:
 
 
 class ExpLauncher(ABC):
+    def __init__(
+        self,
+        experiment_runner: Callable[
+            [Experiment],
+            InfoStats | None,
+        ] = lambda exp: exp.run().trainer_result,
```

Comment:
Anticipating a question here: we can't return a list of  `ExperimentResult` since it will contain an instance of `World` which is not picklable - joblib complains. 

It would also be unnecessary to return all the worlds, they can be quite heavy.

So, instead we return the trainer_result object, which however can be None (but usually is not)